mbs-s0003-20230501/README.md

```diff
@@ -0,0 +1,140 @@
+# Map-based noise simulations based on `scan-s0003` time-domain simulations
+
+Tag: `mbs-s0003-20230501`
+
+## Updates
+
+* 2023-05-09: first release, `lmax=5400` simulations.
+
+## Summary
+
+300 realistic realizations of the map noise, given the single realization of the time-domain simulations. Simulations cover the full LAT footprint to a bandlimit of `lmax=5400`, and utilize the directional wavelet noise model.
+
+## Noise model
+
+This release is based on the `scan-s0003` time-domain simulations, available on `NERSC` at:
```

Comment:
do we have documentation about `scan-s0003`?

Reply:
Reijo is working on a confluence page, but I don't think it's published yet. I can update the documentation when that's available.

Reply:
I made a productdb entry for it and have added the link, but the productdb entry needs the confluence page added still.